cpp/arcticdb/version/local_versioned_engine.cpp

```diff
@@ -249,6 +249,42 @@ std::pair<VersionedItem, FrameAndDescriptor> LocalVersionedEngine::read_datafram
     return std::make_pair(version.value_or(VersionedItem{}), std::move(frame_and_descriptor));
 }
 
+std::pair<VersionedItem, py::object> LocalVersionedEngine::read_descriptor_version_internal(
+        const StreamId& stream_id,
+        const VersionQuery& version_query
+    ) {
+    ARCTICDB_SAMPLE(ReadDescriptor, 0)
+
+    py::object pyobj;
+    auto version = get_version_to_read(stream_id, version_query);
+    if(!version)
+        throw NoDataFoundException(fmt::format("read_descriptor: version not found for stream", stream_id));
```

Comment:
This raises with an error code, right?

Change the error message to not include the word `stream`

Reply:
That's exactly the same one we had previously for read_descriptor. Which is the non-parallel version, sharing now this function with the batch version. Actually, there is a bug in the error message (missing {} for the symbol)